addons/account/models/account_move.py

```diff
@@ -291,6 +291,15 @@ def _sequence_year_range_monthly_regex(self):
     country_code = fields.Char(related='company_id.account_fiscal_country_id.code', readonly=True)
     company_price_include = fields.Selection(related='company_id.account_price_include', readonly=True)
     attachment_ids = fields.One2many('ir.attachment', 'res_id', domain=[('res_model', '=', 'account.move')], string='Attachments')
+    audit_trail_messages = fields.One2many(
```

Comment:
Follow the guidelines for relational fields
```suggestion
    audit_trail_message_ids = fields.One2many(
```